### ISSUE TITLE
All tests fail

### ISSUE THREAD

**Issue by prestonvanloon** [ferranbt/fastssz#6]
```
goos: linux
goarch: amd64
pkg: github.com/ferranbt/fastssz/spectests
BenchmarkMarshalGoSSZ
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x59135b]

goroutine 33 [running]:
github.com/ferranbt/fastssz/spectests.readValidGenericSSZ(0x0, 0x5faf18, 0x52, 0x5c3c40, 0xc000024040, 0xc00027d720, 0x4cae0f, 0xbfbeb46de790ec28)
        /tmp/fastssz/spectests/structs_test.go:406 +0x28b
github.com/ferranbt/fastssz/spectests.BenchmarkMarshalGoSSZ(0xc000120000)
        /tmp/fastssz/spectests/structs_test.go:285 +0x6e
testing.(*B).runN(0xc000120000, 0x1)
        /snap/go/6123/src/testing/benchmark.go:191 +0xe8
testing.(*B).run1.func1(0xc000120000)
        /snap/go/6123/src/testing/benchmark.go:231 +0x57
created by testing.(*B).run1
        /snap/go/6123/src/testing/benchmark.go:224 +0x7d
exit status 2
FAIL    github.com/ferranbt/fastssz/spectests   0.007s
FAIL
```

To reproduce
```
git clone git@github.com:ferranbt/fastssz.git --recursive
go test -v ./spectests/... -run=XXX -bench=.
```

**Comment by prestonvanloon** [comment]
Same issue with the spec tests
```
go test -v ./spectests/... -run TestSpec
=== RUN   TestSpecMinimal
    structs_test.go:406: error unmarshaling JSON: json: cannot unmarshal string into Go value of type spectests.AggregateAndProof
--- FAIL: TestSpecMinimal (0.00s)
=== RUN   TestSpecMainnet
    structs_test.go:406: error unmarshaling JSON: json: cannot unmarshal string into Go value of type spectests.AggregateAndProof
--- FAIL: TestSpecMainnet (0.00s)
FAIL
FAIL    github.com/ferranbt/fastssz/spectests   0.005s
FAIL
```

**Comment by ferranbt** [comment]
Thank you for the issue.
It seems that the spectests in lfs are not downloaded with the recursive flag. I will take a look at this.

**Comment by ferranbt** [comment]
This is fixed with #12 You can use the
```
make ef-tests
```
to locally download the spec tests before running go test.
